<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry"> <rect> <x>0</x> <y>0</y> <width>1100</width> <height>700</height> </rect> </property>
  <property name="windowTitle"> <string>Ù‚Ø§Ø¦Ù…Ø© Ø§Ù„Ø¢Ø«Ø§Ø±</string> </property>
  <property name="layoutDirection"> <enum>Qt::RightToLeft</enum> </property>
  <property name="styleSheet">
   <string notr="true">
    QWidget { font-family: 'Segoe UI', sans-serif; background-color: transparent; }
    
    #pageTitle { font-size: 28px; font-weight: bold; color: #2c3e50; margin-bottom: 10px; }

    /* Cards */
    #topBar, #tableContainer {
        background-color: white;
        border-radius: 15px;
        border-bottom: 4px solid #dce1e6;
        border-right: 1px solid #dce1e6;
    }

    QLineEdit { border: 2px solid #f0f0f0; border-radius: 8px; padding: 8px 15px; background: #fdfdfd; }
    QLineEdit:focus { border: 2px solid #3498db; background: white; }

    QPushButton { font-weight: bold; border-radius: 8px; padding: 10px 20px; border: none; }
    #btnAdd { background-color: #2ecc71; color: white; }
    #btnAdd:hover { background-color: #27ae60; }
    #btnSearch { background-color: #34495e; color: white; }
    #btnSearch:hover { background-color: #2c3e50; }

    /* Table */
    QTableWidget {
        background-color: white;
        border: none;
        border-radius: 10px;
        gridline-color: #f0f0f0;
        selection-background-color: #e3f2fd;
        selection-color: #2c3e50;
    }
    QHeaderView::section {
        background-color: #f8f9fa;
        padding: 10px;
        border: none;
        font-weight: bold;
        color: #7f8c8d;
        border-bottom: 2px solid #e0e0e0;
    }
   </string>
  </property>
  
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing"> <number>20</number> </property>
   <property name="margin"> <number>30</number> </property>
   
   <item> <widget class="QLabel" name="pageTitle"> <property name="text"> <string>Ø¥Ø¯Ø§Ø±Ø© Ø§Ù„Ù…Ø®Ø²ÙˆÙ† Ø§Ù„Ø£Ø«Ø±ÙŠ</string> </property> </widget> </item>
   
   <item>
    <widget class="QFrame" name="topBar">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing"> <number>15</number> </property>
      <property name="margin"> <number>15</number> </property>
      <item> <widget class="QLineEdit" name="searchInput"> <property name="minimumSize"> <size> <width>350</width> <height>40</height> </size> </property> <property name="placeholderText"> <string>ğŸ” Ø§Ø¨Ø­Ø« Ø¨Ø±Ù‚Ù… Ø§Ù„Ø¬Ø±Ø¯ØŒ Ø§Ù„Ø§Ø³Ù…ØŒ Ø£Ùˆ Ø§Ù„ÙƒÙˆØ¯...</string> </property> </widget> </item>
      <item> <widget class="QPushButton" name="btnSearch"> <property name="text"> <string>Ø¨Ø­Ø«</string> </property> </widget> </item>
      <item> <spacer name="hSpacer"> <property name="orientation"> <enum>Qt::Horizontal</enum> </property> </spacer> </item>
      <item> <widget class="QPushButton" name="btnAdd"> <property name="text"> <string>+ Ø¥Ø¶Ø§ÙØ© Ù‚Ø·Ø¹Ø© Ø¬Ø¯ÙŠØ¯Ø©</string> </property> </widget> </item>
     </layout>
    </widget>
   </item>

   <item>
    <widget class="QFrame" name="tableContainer">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="margin"> <number>5</number> </property>
      <item>
       <widget class="QTableWidget" name="artifactsTable">
        <property name="editTriggers"> <set>QAbstractItemView::NoEditTriggers</set> </property>
        <property name="selectionBehavior"> <enum>QAbstractItemView::SelectRows</enum> </property>
        <property name="verticalScrollMode"> <enum>QAbstractItemView::ScrollPerPixel</enum> </property>
        <column> <property name="text"> <string>Ø±Ù‚Ù… Ø§Ù„Ø¬Ø±Ø¯</string> </property> </column>
        <column> <property name="text"> <string>Ø§Ù„ÙƒÙˆØ¯ Ø§Ù„Ø¢Ù„ÙŠ</string> </property> </column>
        <column> <property name="text"> <string>Ø§Ø³Ù… Ø§Ù„Ù‚Ø·Ø¹Ø©</string> </property> </column>
        <column> <property name="text"> <string>Ø§Ù„Ù†ÙˆØ¹</string> </property> </column>
        <column> <property name="text"> <string>Ø§Ù„Ù…Ø§Ø¯Ø©</string> </property> </column>
        <column> <property name="text"> <string>Ø§Ù„Ù…ÙˆÙ‚Ø¹</string> </property> </column>
        <column> <property name="text"> <string>Ø¥Ø¬Ø±Ø§Ø¡Ø§Øª</string> </property> </column>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
</ui>
